tests/5.0/requires/test_requires_atomic_default_mem_order_relaxed.F90

```diff
@@ -0,0 +1,61 @@
+!//===------ test_requires_default_mem_order_relaxed.F90 --------------------------===//
+!
+! OpenMP API Version 5.0 Nov 2018
+!
+! This test checks for support of the atomic_default_mem_order clause on the 
+! requires directive. This clause determines the default memory behavior for
+! atomic constructs. These behaviors are seq_cst, acq_rel, and relaxed.
+! This test checks for relaxed as the memory-order-clause.
+!
+! Adapted from 5.0 OpenMP example acquire_release.3.F90
+!
+!//===----------------------------------------------------------------------===//
+
+#include "ompvv.F90"
+
+PROGRAM test_requires_default_mem_order_relaxed
+  USE iso_fortran_env
+  USE ompvv_lib
+  USE omp_lib
+  implicit none
+
+  !$omp requires atomic_default_mem_order(relaxed)
+
+  OMPVV_TEST_OFFLOADING
+
+  OMPVV_TEST_VERBOSE(test_atomic_relaxed() .ne. 0)
+
+  OMPVV_REPORT_AND_RETURN()
+
+CONTAINS
+  INTEGER FUNCTION test_atomic_relaxed()
+    INTEGER:: x, y, thrd, tmp, errors
+
+    x = 0
+    y = 0
+    errors = 0
+
+    OMPVV_INFOMSG("test_requires_atomic_relaxed")
+
+    !$omp parallel num_threads(2) private(thrd, tmp)
```

Comment:
We need to add a num_threads check here, in case we get only 1 thread. OR omp_set_dynamic(.false.) before parallel.